.github/dependabot.yml

```diff
@@ -12,7 +12,7 @@ updates:
   - package-ecosystem: pip
     directory: "/"
     schedule:
-      interval: daily
+      interval: "monthly"
       time: "04:00"
     open-pull-requests-limit: 10
```

Comment:
@alallema unless you object I think we should bump the `open-pull-request-limit` to 20. It is probably overkill since it would cover every dependency having an update in a month, but I figure it puts us on the safe side and doesn’t hurt anything if it never hits 20 in a month.